meson.build

```diff
@@ -48,13 +48,27 @@ foreach f : check_functions
   endif
 endforeach
 
+check_inlines = ['inline', '__inline', ''],
```

Comment:
This will not build.

```
meson.build:51:42: ERROR:  Expecting eof got comma.
check_inlines = ['inline', '__inline', ''],
```

Also consider inlining this list into the for loop directly, i.e.
```meson
foreach f : ['inline', '__inline', '']
  if cc.compiles(f + ' void func() {}')
    cdata.set('json_inline', f)
    break
  endif
endforeach
```

Reply:
Thanks for reviewing the patch.  I've addressed the trailing comma by inlining the list.

Reply:
Looks good, thank you.